tools/lsp/util.rs

```diff
@@ -14,25 +14,28 @@ use crate::common;
 #[cfg(target_arch = "wasm32")]
 use crate::wasm_prelude::UrlWasm;
 
+pub fn clean_node_range(node: &SyntaxNode) -> TextRange {
```

Comment:
Doc comments for pub functions would be nice. 
(In this case, it is not clear what is meant by "clean")

Also, if it is only used in this module, it doesn't need to be `pub`
If it is used in only one function as a helper, it can be scoped within that function.

Reply:
I extracted it because I need it elsewhere.

Documentation is added now and the name has changed to something that is clearer (I hope;-).